components/applications-service/integration_test/service_groups_filters_test.go

```diff
@@ -223,32 +223,34 @@ func TestDisconnectedStatusFilter(t *testing.T) {
 		require.NoError(t, err)
 
 		req := &applications.ServiceGroupsReq{
-			Filter: []string{"connectedStatus:disconnected"},
+			Filter: []string{"status:disconnected"},
 		}
 		response, err := suite.ApplicationsServer.GetServiceGroups(ctx, req)
 		assert.Nil(t, err)
 		assert.Len(t, response.ServiceGroups, 2)
 	})
-	t.Run("when a status filter and disconnected status filter are both applied and a service group meets both criteria", func(t *testing.T) {
-		servicesToIngest := habServicesMatrix()
-		require.NoError(t, err)
-
-		for _, i := range []int{0, 4, 7, 9} {
-			s := servicesToIngest[i]
-			s.EventMetadata.OccurredAt = oldEventTime
-		}
-		suite.IngestServices(servicesToIngest)
-		defer suite.DeleteDataFromStorage()
-		_, err = suite.ApplicationsServer.MarkDisconnectedServices(300)
-		require.NoError(t, err)
-
-		req := &applications.ServiceGroupsReq{
-			Filter: []string{"connectedStatus:disconnected", "status:critical"},
-		}
-		response, err := suite.ApplicationsServer.GetServiceGroups(ctx, req)
-		assert.Nil(t, err)
-		assert.Len(t, response.ServiceGroups, 1)
-	})
+	// TODO: Re-enable this test when we support multiple selected filters
```

Comment:
is there a card to follow up on this todo?

Reply:
No we don't have one yet, this idea is still in the ideation phase. @apriofrost has talked about making it a cross-automate design change on all tabs.